library/src/main/java/com/schibsted/spain/barista/assertion/BaristaSeekBarAssertions.kt

```diff
@@ -0,0 +1,27 @@
+package com.schibsted.spain.barista.assertion
+
+import android.support.annotation.IdRes
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import com.schibsted.spain.barista.internal.matcher.DisplayedMatchers.displayedWithId
+import com.schibsted.spain.barista.internal.matcher.ProgressBarMatchers.assertProgress
+import com.schibsted.spain.barista.internal.matcher.ProgressBarMatchers.assertProgressIsMax
+import com.schibsted.spain.barista.internal.matcher.ProgressBarMatchers.assertProgressIsMin
+
+object BaristaSeekBarAssertions {
```

Comment:
I thought about it but I wasn't sure if it was going to be intuitive.
I only find this interactions/assertions useful for `SeekBars` and maybe people are not going to know they are available if it says `ProgressBar`.
 In fact until you told me that `SeekBar` was a `ProgressBar` child I didn't know it was hehe.
Although it can be mentioned in README file. What do you think? Should we go with `ProgressBar` in the name and mention that it supports `ProgressBar` and `SeekBar` in the README?

Reply:
I'd go for `setProgress` for the internal method because it sounds more techie (setter and getter) and `changeProgress` for the public one because it implies not only setting the progress, maybe in a future implies more actions. What do you think?

Reply:
@SmasSive I think you commented on the wrong thread :P 
You're probably talking about https://github.com/SchibstedSpain/Barista/pull/182#discussion_r156118773

Reply:
In the case of assertions @jamesbluecrow I think they also make sense for ProgressBar, not just SeekBar. 
I can think of a couple scenarios:
- You have a long wizard where a ProgressBar shows your current page and you want to test that it updates.
- Something like Linkedin profile completion progress, you want to test it shows the right percentage. 

Just some examples to say that in my opinion progress assertion is useful for both SeekBar and ProgressBar. Because of that I would just call them ProgressAssertions, which talks about the concept but not the concrete View.

Reply:
Yes buddy, some mobile issues... Don't try to answer a huge commented PR like this by phone xD

Reply:
Fair enough @Sloy , I think those are valid scenarios. I will update the name today